docs/Quickstart.md

```diff
@@ -0,0 +1,295 @@
+## Quickstart
```

Comment:
Not duplicates (as I understand), I removed Quickstart from main README.md. See comment below.